packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/template.success.tsx

```diff
@@ -1,41 +1,42 @@
+import { Button, Image, Text } from 'blockchain-info-components'
 import {
   CardDetails,
   CardWrapper,
   Child,
   CustomSettingHeader,
   RemoveButton
 } from '../styles'
-import { convertBaseToStandard } from 'data/components/exchange/services'
-import { fiatToString } from 'core/exchange/currency'
 import { FormattedMessage } from 'react-intl'
-import { Icon, Text } from 'blockchain-info-components'
+import { getBankLogoImageName } from '../../../../modals/SimpleBuy/model'
```

Comment:
having this in Simple Buy makes no sense and is just adding to our tech debt.  Can we make a util/service or something?

Reply:
Yes